Source/ACE.Entity/Enum/Properties/PropertyInt.cs

```diff
@@ -2,409 +2,409 @@
 {
     public enum PropertyInt : ushort
     {
-        Undef = 0,
-        ItemType = 1,
-        CreatureType = 2,
-        PaletteTemplate = 3,
-        ClothingPriority = 4,
-        EncumbranceVal = 5, // ENCUMB_VAL_INT,
```

Comment:
Lost note in formatting, please restore so we can track what we changed from client/aclogview